Gemfile

```diff
@@ -2,6 +2,6 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem "rake", "~> 12.0"
```

Comment:
shouldn't we change this to `~> 13.0` instead of removing the version restriction completely?

Reply:
I've done that update on so many repos by now that I know think that this kind of development dependencies shouldn't be locked.

But that's @ko1's call.

Reply:
It should be set `>= 13.0` if we choose the explicitly versioning. We ignore the pessimistic versioning for reducing the maintenance cost.